src/data-layer/TwitchHelix.js

```diff
@@ -7,8 +7,8 @@ const { address: publicIPAddress, port } = publicIP;
 
 class TwitchHelix {
   static appAccess = {
-    accessToken: null,
-    expiresAt: null, // Time in seconds
+    accessToken: "poggers",
```

Comment:
What is this?

Reply:
I've added placeholder data to the twitchhelix file, because it doesnt seem to run locally without it. Node would throw some errors because accessToken would be undefined.